docs/build.sh

```diff
@@ -3,25 +3,13 @@
 # This was adapted from https://github.com/dgraph-io/dgraph/blob/master/wiki/scripts/build.sh
 #
 
-set -e
-
 GREEN='\033[32;1m'
 RESET='\033[0m'
 HOST=https://gqlgen.com
 
-VERSIONS_ARRAY=(
-    'v0.16.0'
-    'v0.15.1'
-    'origin/master'
-    'v0.15.0'
-    'v0.14.0'
-    'v0.13.0'
-    'v0.12.2'
-    'v0.11.3'
-    'v0.10.2'
-    'v0.9.3'
-    'v0.8.3'
-)
+IFS=$'\n' read -r -d '' -a VERSIONS_ARRAY < <(curl -s -H "Accept: application/vnd.github.v3+json" "https://api.github.com/repos/99designs/gqlgen/releases?per_page=20" | jq -r '.[].tag_name')
```

Comment:
Couple of things to note: 
1. This limits to the last 20 versions. I think it's probably enough. We could have more, but those versions risk being very old. 
2. This assumes `jq` & `curl` has been installed on the machine running this script. I suspect that is the case though I am not sure how [render.com](render.com) works
3. This will also change the position of `master` to being at the bottom of the list all the time.